src/components/AnalysisPage/GeoAnalysisBox/GeoAnalysisBox.jsx

```diff
@@ -27,7 +34,10 @@ const GeoAnalysisBox = ({ partId, autoPopAnalysis }) => {
       setSurfaceArea(autoPopAnalysis.surface_area);
       setNumBoundaryEdges(autoPopAnalysis.num_boundary_edges);
       setNumShells(autoPopAnalysis.num_shells);
+      setShowCheckmark(true);
+      setAnalysisComplete(true);
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
This line is just disabling a warning, react wants setShowCheckmark and setAnalysisComplete to be in the dependency array. But they shouldn't be included.